pkg/pii/anonymizer.go

```diff
@@ -13,24 +13,33 @@ type StringAnonymizer[K any] interface {
 	DeanonymizeString(key K, value string) (string, error)
 }
 
-type StructAnonymizer[K any] struct {
+type StructAnonymizer[T any, K any] struct {
```

Comment:
how about `K any, V any`?
or at least `K any, T any` – because we usually put keys first

Reply:
sure, makes sense

Reply:
Please put some doc here what is what. It is public so the users may be confused what are those two generic types.

Reply:
49e0b0ec90e712973aa934a618b64929832fc45d

Reply:
TIL GoLand is not ready to make such changes across the struct